src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php

```diff
@@ -24,16 +24,48 @@
 
 /**
  * @author Andreas Möller <am@localheinz.com>
+ * @author SpacePossum
  */
 final class NativeFunctionInvocationFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
+    /**
+     * @internal
+     */
+    const SET_ALL = '@all';
+
+    /**
+     * Subset of SET_INTERNAL.
+     *
+     * Change function call to functions known to be optimized by the Zend engine.
+     * For details:
+     * - @see https://github.com/php/php-src/blob/php-7.2.6/Zend/zend_compile.c "zend_try_compile_special_func"
+     * - @see https://github.com/php/php-src/blob/php-7.2.6/ext/opcache/Optimizer/pass1_5.c
+     *
+     * @internal
+     */
+    const SET_COMPILER_OPTIMIZED = '@compiler_optimized';
+
+    /**
+     * @internal
+     */
+    const SET_INTERNAL = '@internal';
+
+    private $functionFilter;
```

Comment:
`string[]` ? (guessing...)

Reply:
`static function`, would that be `callable` ? (`gettype()` state `"object"`)

Reply:
yeah, callable ;) good I asked, I wouldn't guess.
for that, nice to document the input/output as well

Reply:
>nice to document the input/output as well

do you maybe have an example for that? not sure how to do this